pimd/pim_cmd.c

```diff
@@ -6861,6 +6878,28 @@ DEFUN (no_ip_pim_packets,
 	return CMD_SUCCESS;
 }
 
+DEFPY (igmp_group_watermark, igmp_group_watermark_cmd,
+       "[no$no] ip igmp watermark-warn [(10-60000)$limit]",
+       NO_STR
+       IP_STR
+       IGMP_STR
+       "Configure group limit for watermark warning\n"
+       "Group count to generate watermark warning\n")
+{
+	PIM_DECLVAR_CONTEXT(vrf, pim);
+	if (no)
+		limit = 0;
+	else {
```

Comment:
if one case has braces so should the other, per kernel style guide

Reply:
This has been removed due to split